docs/extras/modules/data_connection/vectorstores/integrations/azuresearch.ipynb

```diff
@@ -6,14 +6,14 @@
    "source": [
     "# Azure Cognitive Search\n",
     "\n",
-    ">[Azure Cognitive Search](https://learn.microsoft.com/en-us/azure/search/search-what-is-azure-search) (formerly known as `Azure Search`) is a cloud search service that gives developers infrastructure, APIs, and tools for building a rich search experience over private, heterogeneous content in web, mobile, and enterprise applications.\n"
+    "[Azure Cognitive Search](https://learn.microsoft.com/azure/search/search-what-is-azure-search) (formerly known as `Azure Search`) is a cloud search service that gives developers infrastructure, APIs, and tools for building a rich search experience over private, heterogeneous content in web, mobile, and enterprise applications.\n"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Install Azure Cognitive Search SDK"
+    "# Install Azure Cognitive Search SDK"
```

Comment:
Now that the updated Python packages are in PyPi, the `--index-url` argument shouldn't be needed anymore :) You'll still need to specify the version since it's pre-release, but just using `!pip install azure-search-documents==11.4.0b6` should be enough

Reply:
@pablocastro can't we say `!pip isntall azure-search-documents --pre` 
bit cleaner IMHO and defaults to the latest pre-release version.

Reply:
Yep, good point, that's better.